### ISSUE TITLE
DeiT lr question

### ISSUE THREAD

**Issue by King4819** [raoyongming/DynamicViT#42]
Excuse me, I want to ask that: why your training cmds of deit is "--lr 1e-3", but the default settings in your code is "4e-3", Thanks !

**Comment by raoyongming** [comment]
Hi @King4819, our new codebase combined the training code of DeiT and ConvNeXt. The "4e-3" learning rate is for ConvNeXt and Swin models.
